tools/__tasks__/test/javascript/index.js

```diff
@@ -24,7 +24,11 @@ module.exports = {
             task: [
                 {
                     description: 'JS tests',
-                    task: () => exec('jest'),
+                    task: () => exec('jest', null, {
+                        env: {
+                            TZ: 'Europe/London'
```

Comment:
Guessing this is because our old TC servers use the Europe/London time zone?

Reply:
That's my understanding!